services/src/main/java/org/fao/geonet/api/records/MetadataSharingApi.java

```diff
@@ -570,6 +573,10 @@ public SharingResponse getRecordSharingSettings(
         List<GroupPrivilege> groupPrivileges = new ArrayList<>(elGroup.size());
         if (elGroup != null) {
             for (Group g : elGroup) {
+                if (!hasNetworkConfig
+                    && g.getId() == ReservedGroup.intranet.getId()) {
+                    continue;
+                }
```

Comment:
I'm not sure about this change. With this change if you configure later the intranet setting, it seems you'll have to update the privileges of the metadata.

What is the reason of this change? Was it causing some issue?

Reply:
Lot of users are asking what is the Intranet group. Most of the time it is not used. So if you don't set the settings for intranet, this does not display the group in the sharing panel.

![image](https://user-images.githubusercontent.com/1701393/229096212-889c1469-4062-4358-a110-e9cda57e96cd.png)


And if you at some point add intranet settings, then you'll have to change existing records privileges if you want to customize privileges for the intranet group.

Reply:
As long as that is clearly documented, sounds fine.

Reply:
Doc updated @josegar74 https://github.com/geonetwork/doc/pull/234